e2e/cypress/integration/multi_team_and_dm/dm_more_show_user_count_spec.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @multi_team_and_dm
+
+import {getRandomId} from '../../utils';
+
+describe('Multi Team and DM', () => {
+    let testChannel;
+    let testTeam;
+    let testUser;
+
+    const common = 'common';
+    const unique = `${getRandomId(4)}`;
+
+    before(() => {
+        // # Setup with the new team, channel and user
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            testTeam = team;
+            testChannel = channel;
+            testUser = user;
+
+            // # Create 4 test users with prefix 'common'
+            Cypress._.times(4, () => {
+                cy.apiCreateUser({prefix: common}).then(() => {
```

Comment:
Can simply pass the 'common' value if common variable is not to be used elsewhere.
```suggestion
                cy.apiCreateUser({prefix: 'common'}).then(() => {
```